eth-connector/src/lib.rs

```diff
@@ -170,7 +170,6 @@ impl EthConnectorContract {
         account_with_access_right: AccountId,
         owner_id: &AccountId,
     ) -> Self {
-        require!(!env::state_exists(), "Already initialized");
```

Comment:
Any motivation for that?
https://github.com/near/near-sdk-rs/blob/master/examples/fungible-token/ft/src/lib.rs#L70

Reply:
We could add an `expand` to the ci, so the expanded new method looks like [this](https://github.com/aurora-is-near/fast-bridge-protocol/blob/master/near/res/fastbridge_expand.rs#L15728)

Reply:
Here is the expand of eth-connector https://gist.github.com/karim-en/aadaf34281d91be56d5dcb6d8f23d9a4#file-gistfile1-rs-L4619